docs/visualize/controls.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[controls]]
+== Controls Visualization
+
+The Controls visualization enables you to add interactive inputs
+to Kibana dashboards.  You can create two types of inputs:
+a dropdown menu and a radio slider.
+
+To start a *Controls* visualization, open the Visualization application
+and click the *+* button. Scroll to the *Others* section and
+select *Controls*.
+
+[role="screenshot"]
+image::images/controls/controls_in_dashboard.png[]
+
+[[add-input-controls]]
+=== Adding Input Controls
+
+From the *Controls* tab, you can add a dropdown menu
+and range slider to your visualization.
+
+==== Dropdown menu
+
+A dropdown menu allows users to filter content by selecting
+one or more options from a list. The dropdown menu is dynamically populated
+with the results of a terms aggregation.
+
+[role="screenshot"]
+image::images/controls/dropdown_control_editor.png[]
+
+*Control Label*:: The label for the dropdown menu. By default, the
+label is the field name.
+
+*Index Pattern*:: The <<index-patterns,index pattern>> that contains
+the data set to visualize.
+
+*Field*:: The field to use to populate the options list
```

Comment:
Instead of "The field to use to populate the options list and to filter on when users interact with the input.", how about "The field used to populate the options list and filter on when users interact with the input." "to use to" is changed to "used to" and "to filter on" is changed to just "filter on".

Same edits needed under range control